src/Apps/Legion/Models/Wizard.types.ts

```diff
@@ -44,6 +44,12 @@ export interface IViewEntity extends IBaseEntity {
     /** the resolved view model of the `Type` the entity has */
     type: IViewType;
 }
+
+export interface IEntityCounters {
+    discovered: number;
```

Comment:
can we call it "new" intead of "discovered" since that's our semantic in the type?